snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/ext/SnomedExtensionUpgradeTest.java

```diff
@@ -1445,6 +1453,52 @@ public void upgrade27MultipleUpgradesAroundSameConcept() throws Exception {
 		
 	}
 	
+	/**
+	 * This test is to detect and verify a bug in the
+	 * {@link RevisionBranch#difference(RevisionBranch)} (more precisely in the
+	 * RevisionBranchRef#difference method). The problem was that the system used
+	 * SortedSet for {@link RevisionSegment} and built on the fact that branches can
+	 * be always sortable by both time and space. This is true most of the time, but
+	 * when synchronizing old content from earlier branches this might result in
+	 * incorrect conflict reporting behavior due to incorrect branch segment
+	 * diffing. The resulting branch diff contains incorrect branch segments, thus
+	 * the branch compare algorithm fetches irrelevant, already merged commits,
+	 * which are being reported as conflicts in the system (both sides added the
+	 * same component, which is also incorrect).
+	 */
+	@Test
+	public void upgrade25UpgradeExtensionUsingExistingInternationalVersions() {
```

Comment:
There are now two upgrade25 test cases present in this class. Could you please fix the numbers?